conda_forge_tick/utils.py

```diff
@@ -180,10 +180,9 @@ def __enter__(self) -> "LazyJson":
     def __exit__(self, *args: Any) -> Any:
         self._dump()
 
-    def __update__(self, data: dict) -> None:
-        self._load()
-        for key in data.items():
-            self.data[f"{key}"] = data[f"{key}"]
+    def _update(self, data: dict) -> None:
```

Comment:
why did you rename this?

Reply:
It was an idea from CJ: "I don't think this needs to be `__update__` it should be `update`"

Reply:
you have an extra underscore at the front.

Reply:
Yup, I saw with your comment thanks 👍

Reply:
Uhm @beckermr I am getting a problem with this statement `update(self, data: dict)` it's saying that it doesn't match the method base definition of a `MutableMapping` wich should be something as `update(self, data: dict, **kwargs)`. Do you know what can I do to solve this ?

Reply:
You could structure this as:
```python
def update(self, data, **kwargs):
    self._load()
    self._data.update(data, **kwargs)
    self._dump()
```

Reply:
Ok, Thank you very much 👍